### ISSUE TITLE
[CAIA A11y Research - PILOT]: Debt Resolution, Debt Portal (Feb. 2024)

### ISSUE THREAD

**Issue by sara-amanda** [department-of-veterans-affairs/va.gov-team#76534]
# CAIA A11y Support

> [!IMPORTANT]
> **PILOT PREP**: 
> - Type of Assistive Technology that will be primarily utilized in this study: Screen Reader `Types - Needed`
> - Pilot is in Figma, team is [aware](https://github.com/department-of-veterans-affairs/va.gov-team/issues/76030#issuecomment-1948991874).

This is a child ticket of the research ticket [76030](https://github.com/department-of-veterans-affairs/va.gov-team/issues/76030)- created to track work completed for the pilot session.

See task list in this ticket for the `Definition of Done`

## Assignee(s)
-  @eli-oat (co-pilot)

## Description

A CAIA a11y will participate as in a pilot of the [Debt Resolution Team's research study](https://github.com/department-of-veterans-affairs/va.gov-team/issues/76030).

- **Researcher**: The VFS team researcher will test their prototype and conversation guide by conducting the pilot session with a CAIA A11Y.
- **User**: The CAIA A11Y team member will act as a real user in the pilot session.
- **Assistive Tech**: The CAIA A11Y team member will follow the prompts given by the VFS Researcher, while using and demonstrating how the behavior of a `screen reader` on `desktop` and `screen magnifier` on `desktop` would behave.

### Assistive Tech Options

- **Available AT**: `Screen readers` on a `desktop` device is currently the primary source of AT that CAIA A11ys are able to support, in addition to `screen magnification` for user research sessions.
- **Additional Specifications:** If there is a particular `test user`, `browser` or `operating system` you would like us to use, please let us know in advance via this ticket's comments, tagging the assignee - or reach out to @coforma-terry in the CAIA Slack Channel: [#sitewide-content-accessibility-ia ](https://dsva.slack.com/archives/C01K37HRUAH)
    - [ ] @mgleo [If you can let us know the types of technology](https://github.com/orgs/department-of-veterans-affairs/projects/929/views/27?pane=issue&itemId=53141346#:~:text=and%20Device%20Type.-,Table%202%3A%20Technology,-Toggle%20to%20view) that Perigean said will be used, based on the recruitment, that will help us to prep in advance for the pilot. Many thanks! 

## About the Team

<details><summary>Toggle to view information about the team</summary>

- Team name: Debt Resolution
- OCTO-DE product owner: Denise Coveyduc
- Delivery manager: Heather Rienks
- Product manager: Tom Davis
- Researcher: Charlotte Cesana
- Designer/Researcher: Megan Gayle
- Designer: Joe Lee
- FE engineers: Andrew Rodiek, Brandyn Sullins, Aaron Ponce
- BE engineers: Kevin Suarez, Austin Smith
- Product/team Slack channel: #benefits-debt-resolution

</details>

## Pilot Session

> [!TIP]
>  **Prototype is in Figma** 
> [Documentation about Figma's new, experimental accessibility mode.](https://help.figma.com/hc/en-us/articles/7810391964695-Accessible-prototypes-in-Figma)

> [!CAUTION]
> **Debt Resolution is aware and is mitigating any potential disruptions as follows:**
>  "The good thing about this study is that it is two parts so [if screen readers cannot use the prototypes](https://github.com/department-of-veterans-affairs/va.gov-team/issues/76030#issuecomment-1948991874) we will at least get feedback from them during the first part of the session where we are not asking them to use a prototype." 

<details><summary>Toggle to view the scheduling details and links needed</summary>

### Pilot Scheduled
- 2/20/2024 [view the CAIA A11y Capacity Calendar](https://calendar.google.com/calendar/embed?src=c_f982350fc73a8162a1088ab14d00dd37335b2b559a60d6b67eb8021416f81c2a%40group.calendar.google.com&ctz=America%2FNew_York)
### Artifacts
- [Link to prototype for current flow](https://www.figma.com/proto/OiiDTTVTCf8j0GngRg0xxD/Combined-VA-Debt-Portal?type=design&node-id=192-19715&viewport=54,-986,0.25&t=EINFwT8EPigmh19P-0&scaling=min-zoom&starting-point-node-id=192:19715&show-proto-sidebar=1)
- [Link to prototype for revised flow](https://www.figma.com/proto/OiiDTTVTCf8j0GngRg0xxD/Combined-VA-Debt-Portal?type=design&node-id=152-16358&viewport=260,390,0.25&t=EINFwT8EPigmh19P-0&scaling=min-zoom&starting-point-node-id=152:16358&show-proto-sidebar=1)
- [Research plan](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/combined_va_debt_portal/research/exploratory-2024/research-plan.md)
- [Research conversation guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/combined_va_debt_portal/research/exploratory-2024/conversation-guide.md)

</details>

## Pilot Prep

[According to the research plan,](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/combined_va_debt_portal/research/exploratory-2024/research-plan.md) to be sure the website works well for visually impaired people and assistive technologies, the users of this study fit the following criteria:

### Recruitment Screener

<details><summary>Toggle to view the recruitment screener requirements</summary>

[Mentions screen readers, only.](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/combined_va_debt_portal/research/exploratory-2024/research-plan.md#recruitment:~:text=At%20least%203%20of%2020%20people%20who%20use%20a%20screen%20reader)

</details>

## Definition of Done

```[tasklist]
### A11y Tasks
- [x] Coordinate to schedule with a CAIA a11y: Eli available
- [x] Determine date and time for pilot: 2/20/24, 1 p.m. ET
- [x] **Requesting assistive tech devices to be used** `Pending Perigean Info` c/o Team
- [x] **HOLD**:  Set a11y Capacity Calendar `HOLD` on 2/16/2024 for 2/20/2024
- [x] **INVITATION**: Pilot invite needed for CAIA a11y Capacity Calendar: [The person sending the invite would add this to the guest section of the invite.](c_f982350fc73a8162a1088ab14d00dd37335b2b559a60d6b67eb8021416f81c2a@group.calendar.google.com) If they are a CAIA team member they can simply invite the CAIA A11y Capacity Calendar.
- [ ] Review research plan
- [ ] **Review Recruitment for Devices Used**
- [ ] **Receive assistive tech devices to be used
- [ ] **Run** pilot on 2/20/2024: CAIA A11ys to share verbal feedback during the session
- [ ] **Review** and [prototype in Figma](https://staging.va.gov/decision-reviews/submitted-appeal/)
- [ ] Pilot Notes **(Optional)**
```

**Comment by sara-amanda** [comment]
Closing ticket and adding to capacity tracker for 2/20/2024 for @eli-oat 
cc: @coforma-terry

**Comment by sara-amanda** [comment]
Marking GHP closed 2/27/2024 @coforma-terry
